app/pathogen/arbovirus/analyze/page.tsx

```diff
@@ -32,20 +34,32 @@ const VisualizationCard = (props: {
 
 export default function ArboAnalyze() {
   const router = useRouter();
+  const { filteredData } = useContext(ArboContext);
+  const { getNumberOfUniqueValuesForField } = useArboDataInsights();
 
   const renderPlotCardContent = useCallback(({ cardConfigurations }: {cardConfigurations: CardConfiguration[]}) => {
     const allVisualizationInformationWithHeights = addToVisualizationInformation({
       [VisualizationId.ESTIMATE_COUNT_BY_WHO_REGION_AND_ARBOVIRUS]: { height: undefined },
+      [VisualizationId.ESTIMATE_COUNT_BY_UN_REGION_AND_ARBOVIRUS]: { height: undefined },
       [VisualizationId.MEDIAN_SEROPREVALENCE_BY_WHO_REGION]: { height: "h-full" },
       [VisualizationId.CHANGE_IN_MEDIAN_SEROPREVALENCE_OVER_TIME]: { height: "h-full" },
       [VisualizationId.ESTIMATE_COUNT_BY_ARBOVIRUS_AND_ANTIBODY_TYPE ]: { height: undefined },
       [VisualizationId.CUMULATIVE_ESTIMATE_COUNT_OVER_TIME_BY_SAMPLE_FRAME]: { height: "h-full 2xl:h-3/4" },
       [VisualizationId.MEDIAN_SEROPREVALENCE_BY_WHO_REGION_AND_AGE_GROUP]: { height: "h-full" },
     })
 
+    const areLessThanTwoWHORegionsPresentInData = getNumberOfUniqueValuesForField({
+      filteredData: filteredData.filter((dataPoint) => !!dataPoint.whoRegion),
+      fieldName: 'whoRegion'
+    }) < 2;
```

Comment:
This also means that if you filter down to one country or one un region, you'll also see this graph which is an unintended but probably positive side-effect.